.aicoe-ci.yaml

```diff
@@ -6,11 +6,11 @@ check:
   # - thoth-build
 
 # Uncomment following lines to build a public image of this repo
-# build:
-#   build-stratergy: Source
-#   build_source_script: "image:///opt/app-root/builder"
-#   base-image: quay.io/thoth-station/s2i-custom-notebook:latest
-#   registry: quay.io
-#   registry-org: aicoe
-#   registry-project: <CHANGE-ME>
-#   registry-secret: aicoe-pusher-secret
+build:
+  build-stratergy: Source
+  build_source_script: "image:///opt/app-root/builder"
```

Comment:
can you change this to `build-source-script`?

Reply:
this was updated in the [template repo](https://github.com/aicoe-aiops/project-template/blob/master/.aicoe-ci.yaml#L11) later

Reply:
thanks @4n4nd, change made